src/ert/scheduler/scheduler.py

```diff
@@ -275,6 +275,7 @@ async def execute(
             scheduling_tasks.append(asyncio.create_task(self._update_avg_job_runtime()))
 
         sem = asyncio.BoundedSemaphore(self._max_running or len(self._jobs))
+        self._fmok_sem = asyncio.BoundedSemaphore(value=10)
```

Comment:
`value=1` should be enough for everybody

Reply:
Actually, when it is only `1` I can use lock instead :+1: